test/index.html

```diff
@@ -26,6 +26,8 @@
     <script src="./vendor/react-15.4.2.js"></script>
     <script src="./vendor/react-dom-15.4.2.js"></script>
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.js"></script>
```

Comment:
This is just for the initial proof of concept PR. It's importing `babel-core` so the scripts below can handle jsx syntax